version.sbt

```diff
@@ -0,0 +1 @@
+version in ThisBuild := "0.1.0-SNAPSHOT"
```

Comment:
It will be updated to 0.1.0 via the sbt-release plugin once we finish the code review and do the final release.